### ISSUE TITLE
Qual tecnologia front-end usar?

### ISSUE THREAD

**Issue by kvnol** [conhecertudo/conhecertudo.github.io#10]
Seria bom discutirmos qual a melhor tecnologia front-end para que possamos contribuir sem muitas dificuldades e que seja tranquilo dar manutenção e também que seja um projeto escalável.

**Comment by rcmilan** [comment]
Que tal [NextJS](https://nextjs.org/) ? a curva de aprendizagem não é tão grande, tem uma comunidade bastante ativa e é muito simples de fazer deploy com a Vercel

**Comment by dpisati** [comment]
Concordo com o @rcmilan . Next JS eh praticamente React, escala bem e o CI/CD ja esta todo feito pela vercel (horpeda de graca e sem configuracao)
